src/main/resources/view/ResultDisplay.fxml

```diff
@@ -3,7 +3,7 @@
 <?import javafx.scene.control.TextArea?>
 <?import javafx.scene.layout.StackPane?>
 
-<StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/8"
-    xmlns:fx="http://javafx.com/fxml/1">
+<StackPane fx:id="placeHolder" styleClass="result-container" xmlns="http://javafx.com/javafx/8"
```

Comment:
The command result display is not big enough to display everything, nor can it be scrolled. Instead, the words are just cut off.

![image](https://user-images.githubusercontent.com/5585517/95324373-34632800-08d2-11eb-9e1e-ed5b3ff0f470.png)

Reply:
Unable to reliably replicate this. I'll close this comment for now and keep and eye for this bug occurring in the future.